network/wsNetwork.go

```diff
@@ -1162,16 +1161,19 @@ func (wn *WebsocketNetwork) meshThread() {
 		}
 
 		// TODO: only do DNS fetch every N seconds? Honor DNS TTL? Trust DNS library we're using to handle caching and TTL?
-		dnsAddrs := wn.getDNSAddrs()
-		if len(dnsAddrs) > 0 {
-			wn.log.Debugf("got %d dns addrs, %#v", len(dnsAddrs), dnsAddrs[:imin(5, len(dnsAddrs))])
-			wn.dnsPhonebook.ReplacePeerList(dnsAddrs)
-			mp, ok := wn.phonebook.(*MultiPhonebook)
-			if ok {
-				mp.AddPhonebook(&wn.dnsPhonebook)
+		dnsBootstrapArray := wn.config.DNSBootstrapArray(wn.NetworkID)
+		multiPhonebook := MakeMultiPhonebook()
+		for _, dnsBootstrap := range dnsBootstrapArray {
+			dnsAddrs := wn.getDNSAddrs(dnsBootstrap)
+			if len(dnsAddrs) > 0 {
+				wn.log.Debugf("got %d dns addrs, %#v", len(dnsAddrs), dnsAddrs[:imin(5, len(dnsAddrs))])
+				dnsPhonebook := &ThreadsafePhonebook{
+					addrs: dnsAddrs,
+				}
+				multiPhonebook.AddOrUpdatePhonebook(dnsBootstrap, dnsPhonebook)
+			} else {
+				wn.log.Debugf("got no DNS addrs for network %#v", wn.NetworkID)
```

Comment:
1. Change into "Infof"
2. don't use `%#v` everywhere. in this case, I think that `%s` would do.

Reply:
updated